src/fieri/.env

```diff
@@ -1,4 +1,5 @@
 FIERI_FOODCRITIC_FAIL_TAGS=any
 FIERI_FOODCRITIC_TAGS=~FC031 ~FC045
-FIERI_RESULTS_ENDPOINT=http://supermarket.getchef.com/api/v1/cookbook-verisons/evaluation
+FIERI_FOODCRITIC_ENDPOINT=http://supermarket.getchef.com/api/v1/cookbook-verisons/evaluation
```

Comment:
Should the this be along the lines of cookbook-versions/foodcritic_evaluation, or whatever the new route is?